tensorflow/core/tpu/tpu_embedding_optimization_parameters_utils.cc

```diff
@@ -336,7 +336,7 @@ Status LoadOpShapeFunction::operator()(
                     });
   std::vector<shape_inference::ShapeHandle> inputs(user_param_count);
   int input_index = 0;
-  for (int i = 0, iter_limit = state_variable_specs.size(); i < iter_limit;
+  for (int i = 0, end = state_variable_specs.size(); i < end;
        ++i) {
```

Comment:
single line

Reply:
handled in batch 13.

Reply:
Keeping this open until it is solved in any PR

Reply:
updated